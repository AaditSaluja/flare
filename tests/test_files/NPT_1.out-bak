2022-01-27 16:10:13.375212
python 3.8.5 (default, Sep  4 2020, 02:22:02) 
[Clang 10.0.0 ]
numpy 1.20.3
ase 3.21.1
flare 0.2.4

Number of cpu cores: 1
Kernel: ['twobody', 'threebody']
Training points: 0
Cutoffs: {'twobody': 10.0, 'threebody': 6.0}
Number of hyperparameters: 5
Hyperparameter array: [0.64034029 0.16867265 0.0539972  0.4098916  0.05      ]
sig2: 0.64034029 
ls2: 0.16867265 
sig3: 0.0539972 
ls3: 0.4098916 
noise: 0.05 

Uncertainty tolerance: 1.0 times noise hyperparameter 
Timestep (ps): 0.001
Number of frames: 5
Number of atoms: 4
System species: {'H', 'He'}
Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]
Restart: 0 

Previous positions (A):
H        0.3112   -0.2531    0.2194
He       2.2071    2.4322    2.3684
H        5.2068    0.0473   -0.4642
He       7.6764    2.8988    2.1107
--------------------------------------------------------------------------------

Calling DFT...

DFT run complete.
Number of DFT calls: 1
Wall time from start: 0.02 s

*-Frame: 0 
Simulation Time:      0.000 ps 
El            Position (A)                    DFT Force (ev/A)                  Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.2071    2.4322    2.3684       -0.6846    0.0431   -0.0717        0.0000    0.0000    0.0000      -26.8686    6.2185   14.3394
H        5.2068    0.0473   -0.4642        0.2333   -0.1469   -0.1053        0.0000    0.0000    0.0000       21.3546  -11.1210   15.2876
He       7.6764    2.8988    2.1107        0.2457    0.3546   -0.1431        0.0000    0.0000    0.0000       20.8562   -1.1751  -15.1148

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    10.863    10.879    10.890     0.071    -0.014    -0.099

Pressure (GPa): 10.877337 
Temperature (K): 981.99 
Kinetic energy (eV): 0.380797 
Potential energy (eV): -9.330240 
Total energy (eV): -8.949443 

Wall time from start: 0.03 s
Adding atom [0, 1, 2, 3] to the training set.
Uncertainty: [0. 0. 0.]

GP hyperparameters: 
Hyp0 : sig2                           = 0.0000
Hyp1 : ls2                            = 1.8360
Hyp2 : sig3                           = 0.0012
Hyp3 : ls3                            = 0.5063
Hyp4 : noise                          = 0.0925
Likelihood: 5.9342
Likelihood gradient: [ 1.94587429e-03 -5.36581238e-09 -7.58131276e+02  3.56257707e+00
 -2.55329894e+00]
Wall time from start: 0.41 s
--------------------------------------------------------------------------------
-Frame: 1 
Simulation Time:      0.001 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0989    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1795    2.4384    2.3828       -0.6155   -0.0000    0.0194        0.0924    0.0000    0.0000      -28.3417    6.2192   14.3816
H        5.2294    0.0354   -0.4491        0.2755   -0.1835   -0.0089        0.0535    0.0000    0.0000       23.9527  -12.8288   15.0946
He       7.6976    2.8979    2.0955        0.2813    0.2324   -0.0552        0.0755    0.0000    0.0000       21.5316   -0.6102  -15.2414

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     1.216     1.096     1.268     0.214    -0.056    -0.220

Pressure (GPa): 1.193344 
Temperature (K): 1063.56 
Kinetic energy (eV): 0.412426 
Potential energy (eV): -0.126724 
Total energy (eV): 0.285702 

Wall time from start: 0.72 s

Calling DFT...

DFT run complete.
Number of DFT calls: 2
Wall time from start: 0.73 s

*-Frame: 1 
Simulation Time:      0.001 ps 
El            Position (A)                    DFT Force (ev/A)                  Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1795    2.4384    2.3828       -0.7430    0.0507   -0.0673        0.0000    0.0000    0.0000      -28.3417    6.2192   14.3816
H        5.2294    0.0354   -0.4491        0.2769   -0.1587   -0.0853        0.0000    0.0000    0.0000       23.9527  -12.8288   15.0946
He       7.6976    2.8979    2.0955        0.2773    0.3530   -0.1542        0.0000    0.0000    0.0000       21.5316   -0.6102  -15.2414

Periodic cell (A): 
[[10.  0.  0.]
 [ 0.  5.  0.]
 [ 0.  0.  5.]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    10.833    10.874    10.897     0.089    -0.003    -0.111

Pressure (GPa): 10.868103 
Temperature (K): 1063.56 
Kinetic energy (eV): 0.412426 
Potential energy (eV): -9.362877 
Total energy (eV): -8.950451 

Wall time from start: 0.73 s
Mean absolute errors & Mean absolute values
forces mae: 0.0493 eV/A
forces mav: 0.1805 eV/A
type 1 forces mae: 0.0171 eV/A
type 1 forces mav: 0.0868 eV/A
type 2 forces mae: 0.0814 eV/A
type 2 forces mav: 0.2742 eV/A
Adding atom [2, 3, 1, 0] to the training set.
Uncertainty: [0.0535331 0.        0.       ]

GP hyperparameters: 
Hyp0 : sig2                           = 0.0002
Hyp1 : ls2                            = 1.8360
Hyp2 : sig3                           = 0.0012
Hyp3 : ls3                            = 0.5609
Hyp4 : noise                          = 0.0773
Likelihood: 18.8710
Likelihood gradient: [ 1.56563615e+00 -5.81918865e-04  1.72355756e+01 -3.86239368e-02
 -2.20796261e-01]
Wall time from start: 1.40 s
--------------------------------------------------------------------------------
-Frame: 2 
Simulation Time:      0.002 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0994    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1502    2.4444    2.3968       -0.6817   -0.0012    0.0257        0.0940    0.0000    0.0000      -29.9081    6.2187   14.4375
H        5.2542    0.0217   -0.4340        0.3170   -0.2045    0.0133        0.0591    0.0000    0.0000       26.7909  -14.6875   15.1170
He       7.7187    2.8974    2.0798        0.3156    0.2484   -0.0716        0.0810    0.0000    0.0000       22.2532   -0.0309  -15.3958

Periodic cell (A): 
[[ 9.99900069e+00  1.74050712e-04 -1.32700233e-04]
 [ 0.00000000e+00  4.99939519e+00 -9.96300730e-05]
 [ 0.00000000e+00  0.00000000e+00  4.99937907e+00]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     1.538     1.376     1.598     0.265    -0.045    -0.232

Pressure (GPa): 1.504036 
Temperature (K): 1158.83 
Kinetic energy (eV): 0.449372 
Potential energy (eV): -0.176035 
Total energy (eV): 0.273337 

Wall time from start: 1.91 s

Calling DFT...

DFT run complete.
Number of DFT calls: 3
Wall time from start: 1.92 s

*-Frame: 2 
Simulation Time:      0.002 ps 
El            Position (A)                    DFT Force (ev/A)                  Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1502    2.4444    2.3968       -0.8045    0.0575   -0.0620        0.0000    0.0000    0.0000      -29.9081    6.2187   14.4375
H        5.2542    0.0217   -0.4340        0.3257   -0.1715   -0.0651        0.0000    0.0000    0.0000       26.7909  -14.6875   15.1170
He       7.7187    2.8974    2.0798        0.3073    0.3504   -0.1630        0.0000    0.0000    0.0000       22.2532   -0.0309  -15.3958

Periodic cell (A): 
[[ 9.99900069e+00  1.74050712e-04 -1.32700233e-04]
 [ 0.00000000e+00  4.99939519e+00 -9.96300730e-05]
 [ 0.00000000e+00  0.00000000e+00  4.99937907e+00]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    10.808    10.878    10.913     0.111     0.011    -0.128

Pressure (GPa): 10.866268 
Temperature (K): 1158.83 
Kinetic energy (eV): 0.449372 
Potential energy (eV): -9.408126 
Total energy (eV): -8.958754 

Wall time from start: 1.92 s
Mean absolute errors & Mean absolute values
forces mae: 0.0492 eV/A
forces mav: 0.1922 eV/A
type 1 forces mae: 0.0200 eV/A
type 1 forces mav: 0.0937 eV/A
type 2 forces mae: 0.0785 eV/A
type 2 forces mav: 0.2908 eV/A
Adding atom [2, 3, 1, 0] to the training set.
Uncertainty: [0.0590961 0.        0.       ]

GP hyperparameters: 
Hyp0 : sig2                           = 0.0032
Hyp1 : ls2                            = 1.8360
Hyp2 : sig3                           = 0.0011
Hyp3 : ls3                            = 0.5636
Hyp4 : noise                          = 0.0724
Likelihood: 32.8604
Likelihood gradient: [-0.24303182 -0.10291807 10.14178839  0.2366727   0.9820711 ]
Wall time from start: 4.01 s
--------------------------------------------------------------------------------
-Frame: 3 
Simulation Time:      0.003 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.5798    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1191    2.4502    2.4107       -0.7569    0.0015    0.0269        0.5626    0.0000    0.0000      -31.6484    6.2213   14.5038
H        5.2817    0.0062   -0.4188        0.3533   -0.2196    0.0320        0.5468    0.0000    0.0000       30.0047  -16.7218   15.3374
He       7.7401    2.8973    2.0637        0.3709    0.2648   -0.0922        0.5590    0.0000    0.0000       23.0852    0.5872  -15.5963

Periodic cell (A): 
[[ 9.99741241e+00  3.64153947e-04 -2.97743453e-04]
 [ 0.00000000e+00  4.99848915e+00 -2.64152941e-04]
 [ 0.00000000e+00  0.00000000e+00  4.99839089e+00]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     1.814     1.527     1.777     0.285    -0.013    -0.245

Pressure (GPa): 1.705918 
Temperature (K): 1274.96 
Kinetic energy (eV): 0.494405 
Potential energy (eV): 0.071011 
Total energy (eV): 0.565415 

Wall time from start: 4.74 s

Calling DFT...

DFT run complete.
Number of DFT calls: 4
Wall time from start: 4.75 s

*-Frame: 3 
Simulation Time:      0.003 ps 
El            Position (A)                    DFT Force (ev/A)                  Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.1191    2.4502    2.4107       -0.8695    0.0632   -0.0560        0.0000    0.0000    0.0000      -31.6484    6.2213   14.5038
H        5.2817    0.0062   -0.4188        0.3801   -0.1852   -0.0445        0.0000    0.0000    0.0000       30.0047  -16.7218   15.3374
He       7.7401    2.8973    2.0637        0.3353    0.3461   -0.1692        0.0000    0.0000    0.0000       23.0852    0.5872  -15.5963

Periodic cell (A): 
[[ 9.99741241e+00  3.64153947e-04 -2.97743453e-04]
 [ 0.00000000e+00  4.99848915e+00 -2.64152941e-04]
 [ 0.00000000e+00  0.00000000e+00  4.99839089e+00]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    10.783    10.885    10.935     0.137     0.029    -0.148

Pressure (GPa): 10.867437 
Temperature (K): 1274.96 
Kinetic energy (eV): 0.494405 
Potential energy (eV): -9.464431 
Total energy (eV): -8.970026 

Wall time from start: 4.75 s
Mean absolute errors & Mean absolute values
forces mae: 0.0491 eV/A
forces mav: 0.2041 eV/A
type 1 forces mae: 0.0230 eV/A
type 1 forces mav: 0.1016 eV/A
type 2 forces mae: 0.0752 eV/A
type 2 forces mav: 0.3065 eV/A
Adding atom [2, 3, 1, 0] to the training set.
Uncertainty: [0.54681232 0.         0.        ]

GP hyperparameters: 
Hyp0 : sig2                           = 0.0041
Hyp1 : ls2                            = 1.8359
Hyp2 : sig3                           = 0.0010
Hyp3 : ls3                            = 0.5640
Hyp4 : noise                          = 0.0694
Likelihood: 47.4971
Likelihood gradient: [ 1.10811151 -0.28070983 -4.46799976  1.38670563  1.75358532]
Wall time from start: 7.73 s
--------------------------------------------------------------------------------
-Frame: 4 
Simulation Time:      0.004 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.7388    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.0860    2.4558    2.4243       -0.8311    0.0081    0.0218        0.7154    0.0000    0.0000      -33.5722    6.2359   14.5672
H        5.3120   -0.0114   -0.4034        0.3992   -0.2335    0.0531        0.6977    0.0000    0.0000       33.6161  -18.8977   15.7505
He       7.7617    2.8976    2.0470        0.4208    0.2786   -0.1120        0.7136    0.0000    0.0000       24.0466    1.2418  -15.8474

Periodic cell (A): 
[[ 9.99494881e+00  7.23287013e-04 -6.38736453e-04]
 [ 0.00000000e+00  4.99712312e+00 -5.00087608e-04]
 [ 0.00000000e+00  0.00000000e+00  4.99695148e+00]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     2.053     1.757     2.023     0.296     0.028    -0.261

Pressure (GPa): 1.944217 
Temperature (K): 1415.38 
Kinetic energy (eV): 0.548855 
Potential energy (eV): 0.124881 
Total energy (eV): 0.673737 

Wall time from start: 8.73 s

Calling DFT...

DFT run complete.
Number of DFT calls: 5
Wall time from start: 8.74 s

*-Frame: 4 
Simulation Time:      0.004 ps 
El            Position (A)                    DFT Force (ev/A)                  Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.0860    2.4558    2.4243       -0.9402    0.0691   -0.0470        0.0000    0.0000    0.0000      -33.5722    6.2359   14.5672
H        5.3120   -0.0114   -0.4034        0.4406   -0.1997   -0.0237        0.0000    0.0000    0.0000       33.6161  -18.8977   15.7505
He       7.7617    2.8976    2.0470        0.3629    0.3377   -0.1740        0.0000    0.0000    0.0000       24.0466    1.2418  -15.8474

Periodic cell (A): 
[[ 9.99494881e+00  7.23287013e-04 -6.38736453e-04]
 [ 0.00000000e+00  4.99712312e+00 -5.00087608e-04]
 [ 0.00000000e+00  0.00000000e+00  4.99695148e+00]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    10.749    10.892    10.956     0.163     0.059    -0.168

Pressure (GPa): 10.865744 
Temperature (K): 1415.38 
Kinetic energy (eV): 0.548855 
Potential energy (eV): -9.534523 
Total energy (eV): -8.985667 

Wall time from start: 8.74 s
Mean absolute errors & Mean absolute values
forces mae: 0.0475 eV/A
forces mav: 0.2162 eV/A
type 1 forces mae: 0.0253 eV/A
type 1 forces mav: 0.1107 eV/A
type 2 forces mae: 0.0697 eV/A
type 2 forces mav: 0.3218 eV/A
Adding atom [2, 3, 1, 0] to the training set.
Uncertainty: [0.69774078 0.         0.        ]

GP hyperparameters: 
Hyp0 : sig2                           = 0.0125
Hyp1 : ls2                            = 1.8358
Hyp2 : sig3                           = 0.0006
Hyp3 : ls3                            = 0.5650
Hyp4 : noise                          = 0.0679
Likelihood: 62.7795
Likelihood gradient: [ 14.55796228  -1.02064058 691.53484389  -0.84639258 -27.53511406]
Wall time from start: 13.15 s
--------------------
Run complete.
