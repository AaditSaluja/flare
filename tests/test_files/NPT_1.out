2022-01-27 16:12:54.201088
python 3.8.5 (default, Sep  4 2020, 02:22:02) 
[Clang 10.0.0 ]
numpy 1.20.3
ase 3.21.1
flare 0.2.4

Number of cpu cores: 1
Kernel: ['twobody', 'threebody']
Training points: 20
Cutoffs: {'twobody': 10.0, 'threebody': 6.0}
Number of hyperparameters: 5
Hyperparameter array: [1.24917971e-02 1.83576712e+00 6.30676593e-04 5.64987563e-01
 6.78815802e-02]
sig2: 0.012491797115993505 
ls2: 1.8357671229990962 
sig3: 0.0006306765930455734 
ls3: 0.5649875625811804 
noise: 0.06788158020294645 

Uncertainty tolerance: 1.0 times noise hyperparameter 
Timestep (ps): 0.001
Number of frames: 8
Number of atoms: 4
System species: {'H', 'He'}
Periodic cell (A): 
[[ 9.99138634e+00  1.12024111e-03 -1.06497691e-03]
 [ 0.00000000e+00  4.99506936e+00 -8.55976136e-04]
 [ 0.00000000e+00  0.00000000e+00  4.99470642e+00]]
Restart: 5 

Previous positions (A):
H        0.3112   -0.2531    0.2194
He       2.0507    2.4611    2.4376
H        5.3457   -0.0312   -0.3875
He       7.7835    2.8983    2.0295
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
-Frame: 5 
Simulation Time:      0.005 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.0507    2.4611    2.4376       -0.9289    0.0093    0.0220        0.0000    0.0000    0.0000      -35.4324    5.5450   13.9750
H        5.3457   -0.0312   -0.3875        0.4747   -0.2340    0.0758        0.0000    0.0000    0.0000       38.8780  -23.9817   13.7977
He       7.7835    2.8983    2.0295        0.4500    0.2881   -0.1377        0.0000    0.0000    0.0000       25.3719    1.2095  -16.8011

Periodic cell (A): 
[[ 9.99138634e+00  1.12024111e-03 -1.06497691e-03]
 [ 0.00000000e+00  4.99506936e+00 -8.55976136e-04]
 [ 0.00000000e+00  0.00000000e+00  4.99470642e+00]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
     1.017     1.012     1.217     0.228     0.008    -0.181

Pressure (GPa): 1.081918 
Temperature (K): 1595.26 
Kinetic energy (eV): 0.618611 
Potential energy (eV): -0.610668 
Total energy (eV): 0.007943 

Wall time from start: 0.00 s
--------------------------------------------------------------------------------
-Frame: 6 
Simulation Time:      0.006 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       2.0142    2.4667    2.4516       -0.9758    0.0091    0.0173        0.0000    0.0000    0.0000      -37.7293    5.5676   14.0229
H        5.3868   -0.0563   -0.3733        0.5105   -0.2417    0.0949        0.0000    0.0000    0.0000       43.5944  -26.2600   14.6160
He       7.8094    2.8999    2.0126        0.4805    0.2944   -0.1496        0.0000    0.0000    0.0000       26.4942    1.9116  -17.1482

Periodic cell (A): 
[[ 9.99138634e+00  1.12024111e-03 -1.06497691e-03]
 [ 0.00000000e+00  4.99506936e+00 -8.55976136e-04]
 [ 0.00000000e+00  0.00000000e+00  4.99470642e+00]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    -1.102    -0.241     0.192    -0.013    -1.323    -1.114

Pressure (GPa): -0.383740 
Temperature (K): 1795.56 
Kinetic energy (eV): 0.696282 
Potential energy (eV): -0.688242 
Total energy (eV): 0.008040 

Wall time from start: 0.39 s
--------------------------------------------------------------------------------
-Frame: 7 
Simulation Time:      0.007 ps 
El            Position (A)                    GP Force (ev/A)                   Std. Dev (ev/A)                  Velocities (A/ps)       
H        0.3112   -0.2531    0.2194        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000        0.0000    0.0000    0.0000
He       1.9752    2.4720    2.4652       -1.0125    0.0037    0.0150        0.0000    0.0000    0.0000      -40.1169    5.5872   14.0670
H        5.4326   -0.0837   -0.3583        0.5442   -0.2467    0.1126        0.0000    0.0000    0.0000       48.6312  -28.6009   15.6017
He       7.8361    2.9019    1.9947        0.4999    0.2978   -0.1578        0.0000    0.0000    0.0000       27.6696    2.6219  -17.5220

Periodic cell (A): 
[[ 9.99089424e+00  1.20997626e-03 -1.32662010e-03]
 [ 0.00000000e+00  4.99448899e+00 -9.96969461e-04]
 [ 0.00000000e+00  0.00000000e+00  4.99405856e+00]]

Stress tensor (GPa):
       xx        yy        zz        yz        xz        xy
    -1.161    -0.262     0.201    -0.018    -1.429    -1.220

Pressure (GPa): -0.407411 
Temperature (K): 2022.35 
Kinetic energy (eV): 0.784227 
Potential energy (eV): -0.776371 
Total energy (eV): 0.007856 

Wall time from start: 0.78 s
--------------------
Run complete.
